source/connection-options.txt

```diff
@@ -0,0 +1,594 @@
+.. _java-rs-connection-options:
+
+==========================
+Specify Connection Options
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, configure
+
+Overview
+--------
+
+This section describes the MongoDB connection and authentication options
+available in the {+driver-short+}. You can configure your connection using either
+the connection URI or arguments to the ``MongoClient`` constructor.
+
+.. _java-rs-connection-uri:
+
+Using the Connection URI
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you pass a connection URI to the ``MongoClient`` constructor, you can include
+connection options in the string as ``<name>=<value>`` pairs. In the following example,
+the connection URI contains the ``connectTimeoutMS`` option with a value of ``60000``
+and the ``tls`` option with a value of ``true``:
+
+.. code-block:: java
+
+    public class MongoConnection {
+    public static void main(String[] args) {
+        String uri = "mongodb://<hostname>:<port>/?connectTimeoutMS=60000&tls=true";
+        try (MongoClient client = MongoClients.create(uri)) {
+                // Use client here
+            }
+        }
+    }
+
+.. _java-rs-mongo-client-settings:
+
+Using a ``MongoClient``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can pass connection options as arguments to the ``MongoClient`` constructor
+instead of including them in your connection URI.
+Configuring the connection this way makes it easier to
+change settings at runtime and helps you catch errors during compilation.
+The following example shows how to use the ``MongoClient`` constructor to set
+connection options:
+
+.. code-block:: java
+
+    public class MongoConnection {
+        public static void main(String[] args) {
+            String uri = "mongodb://<hostname>:<port>";
+            try (MongoClient client = MongoClients.create(uri, connectTimeoutMS=60000, tls=True) {
```

Comment:
It appears to be Kotlin syntax. In Java, parameter names are not specified directly when calling a method or constructor, making the syntax connectTimeoutMS=60000, tls=True invalid
. 
Instead, parameters can be set through the `MongoClientSettingsBuilder` class, accessible via `MongoClientSettings.builder()`.

Reply:
Thanks for the comment! Based on your comment, we realized that we need a different approach to these docs. We're keeping these docs as connection options for just the connection URI, and making a separate page for connection options using the MongoClientSettingsBuilder/MongoClientSettings.